.github/scripts/release/build-deb.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+
+PRODUCT=$1
+VERSION=$2
+PROFILE=${PROFILE:-production}
+
+cargo install cargo-deb
```

Comment:
```suggestion
cargo install --version 2.7.0 cargo-deb --locked -q
```
I think we should aim for reproducible builds